lib/call-stack-to-nix.nix

```
@@ -6,7 +6,11 @@
  * see also `call-cabal-project-to-nix`!
  */
 { runCommand, nix-tools, pkgs, mkCacheFile, materialize }:
-{ src, stackYaml ? null, ignorePackageYaml ? false, cache ? null
+{ name          ? src.name or null # optional name for better error messages
```

That's an odd alignment? Why is there so much space behind the name, but the other arguments have a single one?
```suggestion
{ name ? src.name or null # optional name for better error messages
```